README.md

```diff
@@ -4,32 +4,31 @@ A set of extensions to [Magit](https://magit.vc) for handling multiple
 repositories simultaneously. This documentation will only make sense
 if you're familiar with Magit.
 
-
 ## Browsing Multiple Repositories
 
-#### [custom variable] multi-magit-selected-repositories
+### \[custom variable\] multi-magit-selected-repositories
```

Comment:
I understand that jumping from h2 to h4 is probably bad style, but h3 looks a bit too large. Not all linters seem to complain about a non-link `[foo]`.